data_upload_DHIS2.py

```diff
@@ -0,0 +1,74 @@
+import requests
+import json
+
+dhis2_username = ''
+dhis2_password = ''
+DHIS2_Test_Server_URL = 'https://ocr.twoca.org/'
```

Comment:
Unless MSF would like to use logins or user authorization down the line, this seems like a good opportunity to use the Streamlit secrets management options, https://docs.streamlit.io/deploy/streamlit-community-cloud/deploy-your-app/secrets-management. If you feel like changing that now, go for it, otherwise we will just keep this in mind for later improvements.